### ISSUE TITLE
[Usable- Get started with channels] "Jump to Bot State Inspection and Update your bot's code" links are not functioning with mouse.

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#1971]
**Actual Result:**
The user will not be able to activate "jump to Bot State Inspection and Update your bot's code" links with mouse as they are not functioning with mouse whereas same can be accessed with keyboard. They both are same page link which scroll the page to their relevant topics.

**Expected Result:**
Every control should be accessible by mouse as well as keyboard. Here, the control is accessible with keys so it should be accessible via mouse also.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to header section and reach to "Help", press enter.
4. Navigate to "Get started with channels" available in Help and activate it.
5. Navigate to "jump to Bot State Inspection and Update your bot's code" links and observe the issue.

**User Impact:**
When a component can't be accessible by mouse the normal user who rely on mouse will never know the purpose of the links as well as user will not be able to use the functionality of the control. 

**Test Environment:**
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Attachment:**
[Links are not functioning by mouse.zip](https://github.com/microsoft/BotFramework-Emulator/files/3819507/Links.are.not.functioning.by.mouse.zip)
